packages/runner/src/types/runner.ts

```diff
@@ -36,6 +36,9 @@ export interface VitestRunnerConfig {
   testTimeout: number
   hookTimeout: number
   retry: number
+  retryDelay?: number
+  retryCondition?: string | ((error: Error) => boolean)
+  retryStrategy?: 'immediate' | 'test-file' | 'deferred'
```

Comment:
Should we instead have everyting inside `retry`? It would allow extending it later, like in https://github.com/vitest-dev/vitest/issues/7834.


```suggestion
  retry: number | {
    count?: number
    delay?: number
    condition?: string | ((error: Error) => boolean)
    strategy?: 'immediate' | 'test-file' | 'deferred'
  }
```

Reply:
Sure, done!